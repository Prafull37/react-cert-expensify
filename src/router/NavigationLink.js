import React from "react";
import {NavLink} from 'react-router-dom'

const NavigationLink=(props)=>(
    <div>
    <NavLink to="/" exact activeClassName="is-active">Dashborad</NavLink>
    <NavLink to="/create" activeClassName="is-active">Add Expense</NavLink>
</div>
)

export default NavigationLink;