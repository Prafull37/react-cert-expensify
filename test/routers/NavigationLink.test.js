import React from "react";
import {shallow} from "enzyme";
// import ReactShallowRenderer from 'react-test-renderer/shallow';

import NavigationLink from "../../src/router/NavigationLink";
import NavLink from 'react-router-dom';

test("test NavigationLink by snapshot",()=>{
  const wrapper=shallow(<NavigationLink/>);
  expect(wrapper).toMatchSnapshot()
})