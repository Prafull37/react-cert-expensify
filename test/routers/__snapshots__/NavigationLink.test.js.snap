// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test NavigationLink by snapshot 1`] = `
<div>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Dashborad
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    to="/create"
  >
    Add Expense
  </NavLink>
</div>
`;
